### ISSUE TITLE
libsvm: remove MacOS specifics install_name_tool

### ISSUE THREAD

**Issue by carlorosati** [Linuxbrew/homebrew-core#1342]
help how do I go about fixing this?

[libsvm.rb](https://github.com/Linuxbrew/homebrew-core/blob/master/Formula/libsvm.rb)

It's using 

```
system "install_name_tool", "-id", "#{lib}/libsvm.2.dylib", "#{lib}/libsvm.2.dylib"
```

**Comment by sjackman** [comment]
I've opened PR #1344 to hopefully resolve this issue.

**Comment by sjackman** [comment]
Fixed by PR #1344. Thanks for the bug report, Carlo.
